File: python/Anggota.py
# class declaration
class Anggota:
    
    __id = 0
    __name = ""
    __bidang = ""
    __partai = ""
    
    def __init__(self, id = 0, name = "", bidang = "", partai = ""):
        self.__id = id
        self.__name = name
        self.__bidang = bidang
        self.__partai = partai
        
    # Getter and Setter. #
    
    # Get id.
    def get_id(self):
        return self.__id
    
    # Set id.
    def set_id(self, id):
        self.__id = id
    
    # Get name.
    def get_name(self):
        return self.__name
    
    # Set name.
    def set_name(self, name):
        self.__name = name
    
    # Get bidang.
    def get_bidang(self):
        return self.__bidang
    
    # Set bidang.
    def set_bidang(self, bidang):
        self.__bidang = bidang
        
    # Get partai.
    def get_partai(self):
        return self.__partai
    
    # Set partai.
    def set_partai(self, partai):
        self.__partai = partai

File: java/Anggota.java
public class Anggota{
    //private atributes.
    private int id;
    private String name;
    private String bidang;
    private String partai;

    /* Constructors */

    public Anggota(){
        this.id = 0;
        this.name = "";
        this.bidang = "-";
        this.partai = "-";
    }

    public Anggota(int id, String name, String bidang, String partai){
        this.id = id;
        this.name = name;
        this.bidang = bidang;
        this.partai = partai;
    }

    /* Getter and Setter. */

    //get name
    public int getId(){
        return this.id;
    }

    //set name
    public void setId(int id){
        this.id = id;
    }

    //get name
    public String getName(){
        return this.name;
    }

    //set name
    public void setName(String name){
        this.name = name;
    }

    //get bidang
    public String getBidang(){
        return this.bidang;
    }
    //set bidang
    public void setBidang(String bidang){
        this.bidang = bidang;
    }

    //get partai
    public String getPartai(){
        return this.partai;
    }

    //set partai
    public void setPartai(String partai){
        this.partai = partai;
    }
}